docs/en/how-to-prepare-for-the-transfer-gateway-update.md

```diff
@@ -0,0 +1,113 @@
+---
+id: how-to-prepare-for-the-transfer-gateway-update
+title: How to Prepare for the Transfer Gateway Update
+---
+
+## Purpose
+
+As part of the Basechain upgrade, we'll be updating the Transfer Gateway. To make the transition as smooth as possible, we've added a new API to `loom-js` for interacting with the Transfer Gateway. This page outlines the changes you'll need to make in order to ensure your DApp continues to work after the Transfer Gateway update goes live.
+
+## 2. Update loom-js to 1.70.1
+
+To use the new API, you have to make sure that you're running `loom-js` version `1.70.1` or later.
+
+In a terminal window, type the following command to see your `loom-js` version:
+
+```bash
+npm ls --depth=0 loom-js
+```
+
+If you're running a version earlier than `1.70.1`, you can update it by entering the following command:
+
+```bash
+npm install loom-js@1.70.1
+```
+
+## 3. Changes to the loom-js API
+
+You must make the following changes so that your codebase correctly uses the new loom-js API. **Otherwise, you won't be able to transfer assets from Loom to Ethereum.**
+
+### 3.1 Use the createEthereumGatewayAsync function to Instantiate the Ethereum Transfer Gateway Used for Withdrawals
+
+The way in which you must instantiate the Ethereum Transfer Gateway used for withdrawals has changed. That said, you need to import the `createEthereumGatewayAsync` function from `loom-js` using something like this:
+
+```js
+import {
+  LocalAddress,
+  CryptoUtils,
+  Address,
+  Contracts,
+  createEthereumGatewayAsync, //Include this line!
+  getMetamaskSigner
+} from 'loom-js'
+```
+
+Then, you can instantiate the gateway as follows:
+
+```js
+async _getEthereumTransferGatewayContract(web3Ethereum) {
+  const networkId = await web3Ethereum.eth.net.getId()
+  console.log("networkId", networkId);
+
+  let version
+  switch (networkId) {
+    case 1: // Ethereum Mainnet
+      version = 1
+      break
+
+    case 4: // Rinkeby
+      version = 2
+      break
+    default:
+      throw new Error('Ethereum Gateway is not deployed on network ' + networkId)
+  }
+
+  const signer = getMetamaskSigner(web3Ethereum.currentProvider) // If you're running `loom-js` in Node.js, change this line to something like `const signer = new OfflineWeb3Signer(rinkeby.web3js, rinkeby.account)
```

Comment:
Can you split the comment on this line into two lines, and put them above this line of code?